web/templates/mark_as_read_disabled_banner.hbs

```diff
@@ -0,0 +1,12 @@
+<p id="mark_as_read_turned_off_content">
+    {{#tr}}
+        Your Zulip app is <z-link>configured</z-link> to not mark messages as read on scroll.
+        {{#*inline "z-link"}}<a href='help/marking-messages-as-read'>{{> @partial-block}}</a>{{/inline}}
+    {{/tr}}
+</p>
+<div id="mark_as_read_controls">
+    <button id="mark_view_read" class="btn btn-warning" title="{{t 'Mark as read' }}">
```

Comment:
Should this `title` be here? It seems redundant and not helpful -- you generally only need titles for buttons that don't have a normal label.

Reply:
Ahh, that makes sense. I've removed it from both unread banners.